teku/src/main/java/tech/pegasys/teku/cli/subcommand/ValidatorClientCommand.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.subcommand;
+
+import com.google.common.base.Throwables;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CompletionException;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.ParentCommand;
+import tech.pegasys.teku.cli.BeaconNodeCommand;
+import tech.pegasys.teku.cli.options.DataOptions;
+import tech.pegasys.teku.cli.options.InteropOptions;
+import tech.pegasys.teku.cli.options.LoggingOptions;
+import tech.pegasys.teku.cli.options.MetricsOptions;
+import tech.pegasys.teku.cli.options.NetworkOptions;
+import tech.pegasys.teku.cli.options.ValidatorClientOptions;
+import tech.pegasys.teku.cli.options.ValidatorOptions;
+import tech.pegasys.teku.storage.server.DatabaseStorageException;
+import tech.pegasys.teku.util.cli.PicoCliVersionProvider;
+import tech.pegasys.teku.util.config.InvalidConfigurationException;
+import tech.pegasys.teku.util.config.NetworkDefinition;
+import tech.pegasys.teku.util.config.TekuConfiguration;
+
+@Command(
+    name = "validator-client",
+    description = "Run a Validator Client that connects to a remote Beacon Node",
+    showDefaultValues = true,
+    abbreviateSynopsis = true,
+    mixinStandardHelpOptions = true,
+    versionProvider = PicoCliVersionProvider.class,
+    synopsisHeading = "%n",
+    descriptionHeading = "%nDescription:%n%n",
+    optionListHeading = "%nOptions:%n",
+    footerHeading = "%n",
+    footer = "Teku is licensed under the Apache License 2.0")
```

Comment:
Can we put a `hidden = true` on this?
And maybe also `aliases = { "vc" }`

Reply:
Done!